generate.php

```diff
@@ -28,14 +28,20 @@
 
 function getModules($client): array
 {
+    // Modules that are present in the repo, but we want to ignore
+    $moduleBlacklist = [
+        'gamification',
+        'ps_emailsmanager',
+        'ps_buybuttonlite',
```

Comment:
sorry if its a dumb question, but why do we need to ignore some modules like these?

Reply:
- ps_buybuttonlite is archived
- gamification and ps_emailsmanager are corp modules  that are still in that repo. @eternoendless is it really needed? Could you try to remove it?